### PR TITLE
Feature/documenting and reformatting

### PR DESCRIPTION
**Pull request by brandonlee503**
Added changes to comments and reformatted some code to make more readable

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,56 +1,66 @@
...
+// Import dependencies
 const express = require('express');
-const path = require('path');
-const favicon = require('serve-favicon');
 const logger = require('morgan');
 const bodyParser = require('body-parser');
 const cors = require('cors');
 
-const courses = require('./routes/courses');
+// =============================================================================
```
Comment (rutgerfarry) [inline app.js]: Not a huge fan of these lines soz

File: app.js [hunk 2]
```diff
@@ -1,56 +1,66 @@
...
+// Returns middleware that only parses urlencoded bodies
 app.use(bodyParser.urlencoded({ extended: false }));
 
+// Initilize express app to use courses route and search route
 app.use('/courses', courses);
-
 // app.use('/search', search);
 
-// catch 404 and forward to error handler
+// =============================================================================
```
Comment (rutgerfarry) [inline app.js]: yeah lets get rid of these

File: routes/courses.js [hunk 3]
```diff
@@ -1,96 +1,123 @@
+// courses.js
+
+// Specify strict mode for 'secure' code
```
Comment (rutgerfarry) [inline routes/courses.js]: lets get rid of these 3. filename is kinda redundant & people should know what 'use_strict' does

File: routes/courses.js [hunk 4]
```diff
@@ -1,96 +1,123 @@
...
 'use_strict';
 
+// Import dependencies and initilize global constants
+
```
Comment (rutgerfarry) [inline routes/courses.js]: importing stuff at the top is pretty standard & people can look up what packages do. lets remove comments about packages

File: routes/courses.js [hunk 5]
```diff
@@ -1,96 +1,123 @@
...
 const moment   = require('moment');
+
+// Library with lots of functions
 const _        = require('underscore');
 
+// PostgreSQL URL and client
 const PG_URL = process.env.DATABASE_URL;
 const client = new pg.Client(PG_URL);
 
+// =============================================================================
```
Comment (rutgerfarry) [inline routes/courses.js]: no lines :smile:

File: routes/courses.js [hunk 6]
```diff
@@ -1,96 +1,123 @@
...
+                abbr: data[0].abbr,
+                credits: data[0].credits,
+                description: data[0].description,
+                sections: [ ],
+            };
+
+        // Disabling JSCS so it wont complain about database column names
+        // jscs:disable
+
+        // ---------------------------------------------------------------------
```
Comment (rutgerfarry) [inline routes/courses.js]: remove lines :-1:

File: routes/search.js [hunk 7]
```diff
@@ -1,29 +1,41 @@
+// search.js
+
+// Specify strict mode for 'secure' code
```
Comment (rutgerfarry) [inline routes/search.js:3]: none of these comments at the top

File: routes/search.js [hunk 8]
```diff
@@ -1,29 +1,41 @@
...
 'use_strict';
 
+// Import dependencies and initilize global constants
 const express  = require('express');
 const router   = express.Router();
 const _        = require('underscore');
 const mongoose = require('mongoose');
 const schemas  = require('../schemas');
 
+// =============================================================================
```
Comment (rutgerfarry) [inline routes/search.js:13]: remove da linez

File: routes/search.js [hunk 9]
```diff
@@ -1,29 +1,41 @@
...
+                console.error(err);
+            } else {
+                res.json(courses);
+            }
+        });
     });
-  });
 });
 
+// =============================================================================
```
Comment (rutgerfarry) [inline routes/search.js:39]: line

File: schemas.js [hunk 10]
```diff
@@ -1,44 +1,58 @@
+// schemas.js
+
+// Import and initilize mongoose for mongodb
 const mongoose = require('mongoose');
 const Schema = mongoose.Schema;
 
+// =============================================================================
```
Comment (rutgerfarry) [inline schemas.js:7]: line

File: schemas.js [hunk 11]
```diff
@@ -1,44 +1,58 @@
...
+    sections: [sectionSchema],
+    updated: { type: Date, default: Date.now },
+    meta: {
     likes: Number,
     dislikes: Number
-  }
+    }
 });
 
+// =============================================================================
```
Comment (rutgerfarry) [inline schemas.js:51]: line

**Comment by brandonlee503** [comment]
Updated dem lines and comments. Everything should seem fine with JSCS rules.  Didn't work on schemas.js or search.js since ones not done yet and ones being removed.

**Pull request merged**